docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+# Amazon Rekognition Detect Text connector blueprint
+
+Amazon Rekognition is a computer vision service that can analyze images and videos. This tutorial shows how to create an OpenSearch connector for Amazon Rekognition's Detect Text API, which can detect and extract text from images.
+
+## Detect text with DetectText API
+
+This tutorial demonstrates how to create an OpenSearch connector for an Amazon Rekognition model that detects text in images using the DetectText API.
+
+Note: This functionality is available in OpenSearch 2.14 or later.
+
+### Step 1: Add the connector endpoint to trusted URLs
+```json
+PUT /_cluster/settings
+{
+  "persistent": {
+    "plugins.ml_commons.trusted_connector_endpoints_regex": [
+      "^https://rekognition\\..*[a-z0-9-]\\.amazonaws\\.com$"
```

Comment:
Can you cut a PR to add this to trusted URL setting?

Reply:
adding to my queue. raised an issue to track this https://github.com/opensearch-project/ml-commons/issues/3308